framework/src/modules/http_api/controllers/blocks.js

```diff
@@ -108,6 +111,67 @@ BlocksController.getBlocks = function(context, next) {
 	});
 };
 
+/**
+ * Gets address by public.
+ *
+ * @private
+ * @param {publicKey} publicKey Public key
+ * @returns {address} address
+ * @todo Replace by Lisk Elements once integrated.
+ */
+__private.getAddressByPublicKey = function(publicKey) {
```

Comment:
avoid using `__private` in the new files. we can just create a function.
Also, lisk-cryptography has this function. can it be used in this version? @MaciejBaj

Reply:
Let's leave the function as is now, a TODO is present to address this later (use Lisk Elements here).